restoredb.sh

```diff
@@ -1,15 +1,20 @@
 #!/bin/bash
 
-PASSWORD="root"
-DATABASE="dockertypo3"
+PASSWORD="1234"
+DATABASE="t3kit"
 USERNAME="root"
 
-mysql -hdb -u"$USERNAME" -p""$PASSWORD"" -e "SHOW DATABASES;"
+if [ -n "${DOCKER}" ]; then
+  DBHOST="db"
+else
+  DBHOST="localhost"
+fi
+mysql -h${DBHOST} -u"$USERNAME" -p""$PASSWORD"" -e "SHOW DATABASES;"
 echo -e "Dropping a database $DATABASE... \r10% "
-mysql -hdb -u"$USERNAME" -p"$PASSWORD" -e "DROP DATABASE $DATABASE;"
+mysql -h${DBHOST} -u"$USERNAME" -p"$PASSWORD" -e "DROP DATABASE $DATABASE;"
 
 echo -e "Installing DB $DATABASE... \r40% "
-mysql -hdb -u"$USERNAME" -p"$PASSWORD" -e "CREATE DATABASE IF NOT EXISTS $DATABASE CHARACTER SET utf8 COLLATE utf8_swedish_ci;"
-mysql -hdb -u"$USERNAME" -p"$PASSWORD" -e "GRANT ALL PRIVILEGES ON $DATABASE.* TO '$USERNAME'@localhost IDENTIFIED BY '$PASSWORD';"
-mysql -hdb -u"$USERNAME" -p"$PASSWORD" $DATABASE < /var/www/shared/db/t3kit.sql
+mysql -h${DBHOST} -u"$USERNAME" -p"$PASSWORD" -e "CREATE DATABASE IF NOT EXISTS $DATABASE CHARACTER SET utf8 COLLATE utf8_swedish_ci;"
+mysql -h${DBHOST} -u"$USERNAME" -p"$PASSWORD" -e "GRANT ALL PRIVILEGES ON $DATABASE.* TO '$USERNAME'@localhost IDENTIFIED BY '$PASSWORD';"
```

Comment:
@heangratha this will now grant privileges for root instead of t3kit user. Could you fix this please?